framework/src/engine/engine.ts

```diff
@@ -55,7 +55,7 @@ const isEmpty = (value: unknown): boolean => {
 
 const emptyOrDefault = <T>(value: T, defaultValue: T): T => (isEmpty(value) ? defaultValue : value);
 
-export class Node {
+export class Engine {
```

Comment:
there are several places in Engine where we are referring to node in the logger like L87, L97, L109.

Reply:
Updating the logging message. 
For the database name, I think `node.db` still make sense because it's storing information of the node.
I think with this setup, (application + engine) creates a client which runs a node